labelbox/client.py

```diff
@@ -1574,6 +1575,48 @@ def get_catalog_slice(self, slice_id) -> CatalogSlice:
         res = self.execute(query_str, {'id': slice_id})
         return Entity.CatalogSlice(self, res['getSavedQuery'])
 
+    def is_feature_schema_archived(self, ontology_id: str,
+                                   feature_schema_id: str) -> bool:
+        """
+        Returns true if a feature schema is archived in the specified ontology, returns false otherwise.
+
+        Args:
+            feature_schema_id (str): The ID of the feature schema
+            ontology_id (str): The ID of the ontology
+        Returns:
+            bool
+        """
+
+        ontology_endpoint = self.rest_endpoint + "/ontologies/" + urllib.parse.quote(
+            ontology_id)
+        response = requests.get(
+            ontology_endpoint,
+            headers=self.headers,
+        )
+
+        if response.status_code == requests.codes.ok:
+            feature_schema_nodes = response.json()['featureSchemaNodes']
+            tools = feature_schema_nodes['tools']
+            classifications = feature_schema_nodes['classifications']
+            relationships = feature_schema_nodes['relationships']
+            feature_schema_node_list = tools + classifications + relationships
+
+            is_feature = lambda f: f['featureSchemaId'] == feature_schema_id
```

Comment:
This can be simplified to:
```
filtered_feature_schema_nodes = [feature_schema_node for feature_schema_node in feature_schema_node_list if feature_schema_node['featureSchemaId'] == feature_schema_id]
if filtered_feature_schema_nodes:
  return bool(filtered_feature_schema_nodes[0]['archived']
```

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/927/commits/5f4ca70a5b6b54939efdd2aac9683921c07410c1)